docs/docs/getting-started/v3-migration.md

```diff
@@ -66,6 +66,7 @@ A number of changes were made to the configuration options passed to the `Chart`
 
 * `hover.animationDuration` is now configured in `animation.active.duration`
 * `responsiveAnimationDuration` is now configured in `animation.resize.duration`
+* Polar area `elements.arc.angle` option now returns degrees instead of radians.
```

Comment:
`return` combined with `option` does not sound right to me. Is the behavior now different with other charts using Arc?

Reply:
I updated the wording to match the others. This doesn't affect other charts that use the `Arc` element as only the polar area controller reads the setting.